inference-engine/src/mkldnn_plugin/nodes/mkldnn_crop_node.cpp

```diff
@@ -1,198 +0,0 @@
-// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Why it was not removed by rebasing on master?

Reply:
i have no idea
but on master this files is absent